static/monorail-2.0-sb.yaml

```diff
@@ -118,7 +118,48 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
-
+  /notification:
+    x-swagger-router-controller: notification
+    post:
+      operationId: notificationPost
+      summary: |
+        post a notification
+      description: |
+        post a notification
+      parameters:
+        - name: taskId
+          in: query
+          description: |
+            node identifier
+          required: false
+          type: string
+        - name: data
+          in: query
+          description: |
+            node identifier
+          required: false
```

Comment:
required: true, required: true, since you add assertion logic if the data is missing.